test_runner/regress/test_lsn_mapping.py

```diff
@@ -111,13 +108,56 @@ def test_lsn_mapping(neon_env_builder: NeonEnvBuilder):
         # make sure that we return the minimum lsn here at the start of the range
         assert Lsn(result["lsn"]) >= last_flush_lsn
 
+
+#
+# Test if cancelled pageserver get_lsn_by_timestamp request is correctly handled.
+#
+# Added as an effort to improve error handling and avoid full anyhow backtrace
+# https://github.com/neondatabase/neon/pull/7949
+#
+def test_get_lsn_by_timestamp_cancelled(neon_env_builder: NeonEnvBuilder):
+    env = neon_env_builder.init_start()
+
+    tenant_id, _ = env.neon_cli.create_tenant(
```

Comment:
I don't think all these configurations are necessary

Reply:
You can take a look at `test_aux_v2_config_switch` on how to retrieve default tenant+timeline ID